build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,223 @@
+      inputs:
+        provisionator_uri: $(provisionator-uri)
+        github_token: $(GitHub.Token)
+        provisioning_script: $(System.DefaultWorkingDirectory)/external/monodroid/build-tools/provisionator/profile.csx
+        provisioning_extra_args: -vv
+
+    - script: make prepare-deps V=1 CONFIGURATION=$(Configuration) MSBUILD_ARGS="$(AutoProvisionArgs)"
+      displayName: make prepare-deps
+
+    - script: make prepare jenkins V=1 CONFIGURATION=$(Configuration) MSBUILD_ARGS="$(AutoProvisionArgs) /p:BundleRootPath=$(System.DefaultWorkingDirectory)"
+      displayName: make prepare jenkins
+
+    - script: make create-installers V=1 CONFIGURATION=$(Configuration)
+      displayName: make create-installers
+
+    - script: |
+        mkdir -p $(System.DefaultWorkingDirectory)/unsigned-installers
+        cp bin/Build$(Configuration)/*.vsix $(System.DefaultWorkingDirectory)/unsigned-installers
+        cp bin/Build$(Configuration)/*.pkg $(System.DefaultWorkingDirectory)/unsigned-installers
+      displayName: copy build artifacts
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload unsigned installers
+      inputs:
+        artifactName: unsigned
+        targetPath: $(System.DefaultWorkingDirectory)/unsigned-installers
+
+    - task: MSBuild@1
+      displayName: package build results
+      inputs:
+        solution: build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj
+        configuration: $(Configuration)
+        msbuildArguments: /t:ZipBuildStatus /p:ResultPackageOutputPath=$(Build.ArtifactStagingDirectory)
+      condition: always()
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload artifacts
+      inputs:
+        artifactName: mac-build-results
+        targetPath: $(Build.ArtifactStagingDirectory)
+      condition: always()
+
+- stage: win_build_test
+  displayName: Win
+  dependsOn: prepare
+  variables:
+    Configuration: Release
+  jobs:
+  - job: build_test
+    displayName: Build and Test
+    pool: VSEng-MicroBuildVS2017
+    timeoutInMinutes: 360
+    cancelTimeoutInMinutes: 5
+    variables:
+      _AndroidDesignTimeResDirIntermediate: .
+      CleanAllFiles: False
+      CleanOnNuGetPackagesChange: False
+      NuGet.Show.Stack: true
+      NUnit.Extra: --trace=Verbose
+    steps:
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(BundleArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)
+
+    - task: MSBuild@1
+      displayName: msbuild Xamarin.Android /t:Prepare
+      inputs:
+        solution: Xamarin.Android.sln
+        msbuildVersion: 15.0
+        msbuildArguments: /t:Prepare /bl:$(Build.ArtifactStagingDirectory)\prepare.binlog /p:BundleRootPath=$(System.DefaultWorkingDirectory)
+
+    - task: MSBuild@1
+      displayName: msbuild Xamarin.Android /t:Build
+      inputs:
+        solution: Xamarin.Android.sln
+        msbuildVersion: 15.0
```

Comment:
Same here.

Reply:
Removed, when no value is provided we'll default to 'latest'.